fuel/downloaders/base.py

```diff
@@ -55,6 +69,9 @@ def download(url, file_handle, chunk_size=1024):
         URL to download.
     file_handle : file
         Where to save the downloaded URL.
+    fd : open file handle, optional
```

Comment:
The reason for that is I re-used the same name as in `progressbar2`, as well as the same default value. I have nothing against giving it a better name and a saner default. Ideas?